packages/server/lib/project.js

```diff
@@ -99,6 +101,23 @@ class Project extends EE {
 
         return scaffold.plugins(path.dirname(cfg.pluginsFile), cfg)
       }
+    }).then((cfg) => {
+      try {
+        const tsPath = resolve.sync('typescript', {
```

Comment:
this PR has already been merged, but it is a bad idea to ever use `sync` functions which read from the file system. this should use the async variant that does async `fs` calls under the hood. this avoids potential `EMFILE` errors.